packages/cf.js/considerations-for-0-0-2.md

```diff
@@ -0,0 +1,13 @@
+# missing features of v0.0.2
+
+- need a way to provide a `timeout` during the install flow
+    - argument to `proposeInstall`?
+    - attribute of the manifest.json
+- currently, the user's address is never being set
+    - provide as an argument: `new Client(nodeProvider, address)`
+    - fetch it from the wallet through an `async init` method?
+- do dapp's specify the deposit? if so, `proposeInstall` and `install` need a deposit arg
+- should the `terms` be set during `install` instead of `proposeInstall`, since we won't know the full limit of the terms until both parties have deposited?
+- should the dapp be able to provide an initial state during `install`? or do we expect dapps to immediately perform an `updateState`?
```

Comment:
Definitely need to pass an initial state to install().  Good catch

Reply:
What's the rationale behind providing an initial state?

Is this motivated by recovering from a crash? Otherwise apps should be instantiated with a default state, no?

Reply:
Yeah I can see why that'd be required for Ethmo. Ethmo's default state is effectively the input to the app, and I can see other apps having the same dynamic.

Though I think some others apps wouldn't need it.

For example, if I'm instantiating a game of Chess, it'd be annoying for me to construct an initial state for it.

Reply:
I wonder if it might be worth having a `default_initial_state` attribute in the manifest? So that dapps could simply fetch/use that as a fallback. Regardless, in the case of the chess game, you'd have to define it somewhere.

Reply:
I think you always need it @cf19drofxots, even for something like Chess. You need to _at least_ provide the payout addresses in the initial state, otherwise money you deposit into an app is lost forever.